apiserver/pkg/versioneddataset/versioned_dataset.go

```diff
@@ -124,13 +125,24 @@ func VersionFiles(ctx context.Context, _ client.Client, input *generated.Version
 		return objectInfoList[i].LastModified.After(objectInfoList[j].LastModified)
 	})
 
+	existMap := make(map[string]int)
```

Comment:
If you just determine whether the key exists, you can define it directly as follows
```
existMap := make(map[string]struct{})
```